pom.xml

```
@@ -85,7 +85,12 @@
         <spark.compat.version>3.4</spark.compat.version>
         <spark.major.version>3</spark.major.version>
         <log4j.version>2.19.0</log4j.version>
-        <graphframe.version>0.8.3-spark3.4</graphframe.version>
+
+
+        <graphframes.groupId>io.graphframes</graphframes.groupId>
+        <graphframes.artifactId>graphframes-spark${spark.major.version}_${scala.compat.version}</graphframes.artifactId>
+        <!--There is no 3.4 version of GraphFrames, so we use 0.9.0-spark3.5-->
+        <graphframes.version>0.9.0-spark3.5</graphframes.version>
```

Versioning was fixed so this can just be 
```suggestion
        <graphframes.version>0.9.2</graphframes.version>
```